requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
git add requirements-dev.in

Reply:
It didn't change... only changes to the generated file because of the `-c requirements.txt`

Reply:
Gotcha, thanks!

You need to use python 3.8 to generate this though? Don't think we've moved to 3.9 yet.